code/modules/urist/modules/jungle/jungle_animals.dm

```diff
@@ -460,20 +464,23 @@
 	icon_living = "cougar"
 	icon_dead = "cougar_dead"
 	icon_gib = "cougar_dead"
+	skin_material = MATERIAL_SKIN_FUR
 
 /mob/living/simple_animal/hostile/huntable/bear/grey
 	desc = "A big scary bear, probably best to stay away"
 	icon_state = "greybear"
 	icon_living = "greybear"
 	icon_dead = "greybear_dead"
 	icon_gib = "greybear_dead"
+	skin_material = MATERIAL_SKIN_FUR_BLACK
```

Comment:
does this make more sense as `MATERIAL_SKIN_FUR_GREY`?

Reply:
player convenience, we dont have any grey pelts so at least with black fur they can make a bear pelt from a grey bear.